client/components/ReviewsWidget.jsx

```diff
@@ -16,24 +16,29 @@ export default class ReviewsWidget extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      reviewData: {
-        avgRating: 0,
-        itemName: '',
-        mainImage: '',
-        reviewsArray: [],
-      },
+      storeCount: 0,
+      rating: 0,
+      productCount: 0,
+      reviewList: [],
+      reviewPictures: [],
     };
   }
 
   componentDidMount() {
     // const queryString = window.location.pathname;
     // var id = url.substring(url.lastIndexOf('/') + 1);
     const id = 2;
```

Comment:
assuming this is placeholder for future code that gets the correct id

Reply:
Yes that's one way to get the id from the URL.